tests/TestCase/Utility/SecurityTest.php

```diff
@@ -70,6 +70,20 @@ public function testHash()
     }
 
     /**
+     * testInvalidHashTypeException
+     *
+     * @return void
+     */
+    public function testInvalidHashTypeException() {
+        try {
+            Security::hash('test', 'doesnotexist', false);
+            $this->fail('Expected \RuntimeException');
+        } catch (\RuntimeException $e) {
```

Comment:
Is it common to do FQNS in tests?

Reply:
I still got a sniffer to auto fix this globally :)

Reply:
Why is it not part of the cakephp tests running on GH :)?

Reply:
Because no one ported it over yet.